app/src/main/java/com/gdg/android/MainActivity.kt

```diff
@@ -97,44 +99,99 @@ fun Greeting(name: String, depart: String, modifier: Modifier = Modifier) {
         )
 
         Text(
-            text = "24-2 수강 과목",
+            text = stringResource(R.string._24_2),
             modifier = modifier.padding(top = 10.dp),
             color = Color.Black,
         )
+
+        LazyColumn {
+            items(subjects) { subject ->
+                Text(
+                    modifier = Modifier.padding(vertical = 16.dp, horizontal = 5.dp),
+                    text = subject
+                )
+                HorizontalDivider(
+                    modifier = Modifier.fillMaxWidth(),
+                    thickness = 1.dp,
+                    color = Color.LightGray
+                )
+            }
+        }
     }
 }
 
+@Composable
+fun SignInScreen(navController: NavController) {
+    val (department, setDepartment) = remember {
+        mutableStateOf("")
+    }
+
+    val (name, setName) = remember {
+        mutableStateOf("")
+    }
+
+    val context = LocalContext.current
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Color.White),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center
+    ) {
+        Text("안녕하세요, 여러분", fontSize = 30.sp)
+        Spacer(modifier = Modifier.height(16.dp))
+
+        Text("학부", fontSize = 20.sp)
+        TextField(value = department, onValueChange = setDepartment,
+            placeholder = {
+                Text(
+                    "학부를 입력해주세요",
```

Comment:
속성명을 먼저 적어주는게 보기에도 편합니다!!
```suggestion
                    text = "학부를 입력해주세요",
```